.gitignore

```diff
@@ -53,3 +53,8 @@ Thumbs.db
 
 #h2
 *.db
+/console/src/main/resources/static/css/
```

Comment:
why ignore these files?

Reply:
Because these files are generated together in the maven package.